css/components/content-wrapper/_content-wrapper.scss

```diff
@@ -1,32 +1,33 @@
 /* CONTENT WRAPPER */
-.content-wrapper__aside {
-    display: none;
-}
+.content-wrapper {
+    &__aside {
+        display: none;
 
-@media screen and (min-width: 800px) {
-    .content-wrapper {
-        display: flex;
-        margin: 0 -15px;
+        @media screen and (min-width: 800px) {
+            padding: 0 15px;
+            display: block;
+            flex: 1;
+        }
     }
 
-    .content-wrapper__main,
-    .content-wrapper__aside {
-        padding: 0 15px;
-    }
-    
-    .content-wrapper__main {
-        width: 70%;
+    &__main {
+        @media screen and (min-width: 800px) {
+            padding: 0 15px;
+            width: 70%;
+        }
     }
 
-    .content-wrapper__aside {
-        display: block;
-        flex: 1;
+    &__aside-user {
+        @media screen and (min-width: 800px) {
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+            margin-bottom: 30px;
+        }
     }
 
-    .content-wrapper__aside-user {
+    @media screen and (min-width: 800px) {
```

Comment:
daj to wyżej, nad &__aside

najpierw stylujemy glowny element razem z media queries, a potem jego elementy

Reply:
poszedł commit